### DISCUSSION TITLE
Will there be a way to transfer an existing project to the new project feature

### DISCUSSION THREAD

**Discussion by andreacab2** [community/community#6606] [Projects and Issues]
Hi there, 

I have tried to use the same github actions in our .github/workflows folder for the beta project to just copy new data from our current project to the new one with the goal of slowly converging both to the same state. 

However, even with the same configuration and tokens, when the actions run and retrieve the list of projects, the beta project is not present in the list received from the gh api.

Is that expected ? If it is how do I retrieve the list of new projects (currently beta) ?

Additionally, I think it would be great to have a way to migrate existing projects to the new one.

**Comment by galargh** [comment]
There is an earlier discussion on migrating to projects (beta), see https://github.com/github/feedback/discussions/6141. 

In particular, https://github.com/github/feedback/discussions/6141#discussioncomment-2322825 might be of interest to you - I created a tutorial on how I approached the migration myself.
